src/smt/theory_str_noodler/aut_assignment.cpp

```diff
@@ -208,7 +208,7 @@ namespace smt::noodler {
     }
 
     void AutAssignment::replace_dummy_with_new_symbol() {
-        mata::Symbol new_symbol = regex::Alphabet(alphabet).get_unused_symbol();
+        mata::Symbol new_symbol = alphabet.size() == 0 ? 92 : regex::Alphabet(alphabet).get_unused_symbol();
```

Comment:
If the alphabet size is 0, then there cannot be a dummy symbol, I would just return.

Reply:
Yes, right